src/view/com/posts/PostFeed.tsx

```diff
@@ -599,15 +604,14 @@ let PostFeed = ({
         return <TrendingInterstitial />
       } else if (row.type === 'interstitialTrendingVideos') {
         return <TrendingVideosInterstitial />
+      } else if (row.type === 'fallbackMarker') {
+        // HACK
+        // tell the user we fell back to discover
+        // see home.ts (feed api) for more info
+        // -prf
+        return <DiscoverFallbackHeader />
       } else if (row.type === 'sliceItem') {
         const slice = row.slice
-        if (slice.isFallbackMarker) {
```

Comment:
this wasn't working because the row for this slice never got pushed (because it had no items). the fix it to make it a first-class row type now that is easy to do.